src/main/java/aquality/selenium/forms/FormParent.java

```diff
@@ -0,0 +1,501 @@
+     */
+    protected <T extends IElement> List<T> findChildElements(By childLoc, ElementType elementType, ElementsCount count) {
+        return findChildElements(childLoc, elementType, ElementState.DISPLAYED, count);
+    }
+
+    /**
+     * Finds child elements by their locator relative to parent element.
+     *
+     * @param childLoc    Locator of child elements relative to its parent.
+     * @param elementType type of the element to be obtained
+     * @param state       Visibility state of child elements.
+     * @return List of child elements.
+     */
+    protected <T extends IElement> List<T> findChildElements(By childLoc, ElementType elementType, ElementState state) {
+        return findChildElements(childLoc, elementType, state, ElementsCount.ANY);
+    }
+
+    /**
+     * Finds child elements by their locator relative to parent element.
+     *
+     * @param childLoc    Locator of child elements relative to its parent.
+     * @param elementType type of the element to be obtained
+     * @param state       Visibility state of child elements.
+     * @param count       Expected number of elements that have to be found (zero, more then zero, any).
+     * @return List of child elements.
+     */
+    protected <T extends IElement> List<T> findChildElements(By childLoc, ElementType elementType, ElementState state,
+                                                             ElementsCount count) {
+        return findChildElements(childLoc, null, elementType, state, count);
+    }
+
+    /**
+     * Finds displayed child elements by their locator relative to parent element.
+     *
+     * @param childLoc    Locator of child elements relative to its parent.
+     * @param name        Child elements name.
+     * @param elementType type of the element to be obtained
+     * @return List of child elements.
+     */
+    protected <T extends IElement> List<T> findChildElements(By childLoc, String name, ElementType elementType) {
+        return findChildElements(childLoc, name, elementType, ElementsCount.ANY);
+    }
+
+    /**
+     * Finds displayed child elements by their locator relative to parent element.
+     *
+     * @param childLoc    Locator of child elements relative to its parent.
+     * @param name        Child elements name.
+     * @param elementType type of the element to be obtained
+     * @param count       Expected number of elements that have to be found (zero, more then zero, any).
+     * @return List of child elements.
+     */
+    protected <T extends IElement> List<T> findChildElements(By childLoc, String name, ElementType elementType,
+                                                             ElementsCount count) {
+        return findChildElements(childLoc, name, elementType, ElementState.DISPLAYED, count);
+    }
+
+    /**
+     * Finds child elements by their locator relative to parent element.
+     *
+     * @param childLoc    Locator of child elements relative to its parent.
```

Comment:
could you check please existence of "Type of the target elements." for the methods

Reply:
fixed